src/monattack.cpp

```diff
@@ -4860,3 +4860,123 @@ bool mattack::dodge_check(monster *z, Creature *target){
     }
     return false;
 }
+
+void mattack::ink_jet( monster *z, int index ) {
+
+    if (!z->can_act()) {
+        return;
+    }
+
+    int jet_range =  4;
+
+    // TODO: make the creature utilise this ability then escaping the threat
+    Creature *target = z->attack_target();
+
+    if (target == nullptr || rl_dist(z->pos(), target->pos()) > jet_range || !z->sees(*target)) {
+        return;
+    }
+
+    std::vector<tripoint> line = g->m.find_clear_path( z->pos(), target->pos() );
+
+    z->reset_special( index );
+
+    z->moves -= 50;
+
+    const game_message_type msg_type = target->attitude_to( g->u ) == Creature::A_FRIENDLY ? m_bad : m_warning;
+
+    target->add_msg_player_or_npc( msg_type, 
+        _("The %s squirts a jet of black ink at your eyes!"),
+        _("The %s squirts a jet of black ink at <npcname>!'s eyes"),
```

Comment:
You could omit the part about eyes and just have "squirts ink at name-here". Some monsters don't have eyes (for example, blobs), and it's pretty obvious that the ink squirt is aimed at eyes (or cameras).